### DISCUSSION TITLE
Does not work with single command:

### DISCUSSION THREAD

**Discussion by sergei-dyshel** [oclif/plugin-test-esbuild-single#181] [Q&A]
I'm trying to apply this approach to CLI with single command. So I created a single `Command` :
```ts
// index.js

import {Command } from '@oclif/core'

class MyCommand extends Comman {
  ....
  async run() {
  ...
   }
}
```
```jsonc
# package.json
"oclif": {
  "bin": "bundle",
	"commands": {
	        "strategy": "single",
	        "target": "./dist/index.js"
    }
}
```

However when running without arguments, I"m getting:
```
USAGE
  $ bundle
```

Without any args/flags.

Am I doing something wrong?

**Comment by git2gus** [comment]
Git2Gus App is installed but the `.git2gus/config.json` doesn't exist.
